### DISCUSSION TITLE
The plugin for 'Patches' has not been loaded into Immer.

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3586] [Q&A] [answered]
Unable to use sample code basic tree in my application. 
![image](https://user-images.githubusercontent.com/19658624/167048720-0dcdba5f-7dba-401a-99ca-74278af133e3.png)
I've tried to call enablePatches() in index.ts, but not work.

**Comment by grigasp** [comment]
@roluk @saskliutas

**Comment by roluk** [comment, marked as answer]
This error usually indicates that `UiComponents.initialize()` hasn't been called.
